lib/theme_check/language_server/completion_providers/object_completion_provider.rb

```diff
@@ -22,12 +22,14 @@ def partial(variable_lookup)
 
       private
 
-      def object_to_completion(object)
-        content = ShopifyLiquid::Documentation.object_doc(object)
+      def object_to_completion(object_name)
```

Comment:
In `def completions`, you call `ShopifyLiquid::Object.labels` which runs `SourceIndex.objects.map(&:name)`.
And then here, you call `ShopifyLiquid::SourceIndex.objects.find`.

Why not call `SourceIndex.objects` directly then, `.select { |w| w.name.start_with?(partial(variable_lookup)) }`, and give the whole object as a parameter to `object_to_completion`, so that no `.find` operation needs to be made?

Reply:
Excellent catch!! 🔥🔥🚀

Now, we're no longer missing `discount`:

![image](https://user-images.githubusercontent.com/1079279/207901638-c1e22905-2977-4e8f-bce6-bbafd9f78b30.png)